components/arm/universalrobots/ur.go

```diff
@@ -509,6 +510,10 @@ func (ua *urArm) moveToJointPositionRadians(ctx context.Context, radians []float
 	}
 }
 
+func (ua *urArm) Kinematics(ctx context.Context) (referenceframe.Frame, error) {
+	return nil, errors.New("urArm arm.Kinematics is unimplemented")
```

Comment:
Same thing here, the implementation is super simple, just the same thing we are doing in the `ModelFrame` function